src/main/java/com/starkinfra/utils/HttpClient.java

```diff
@@ -23,7 +23,7 @@ public static synchronized ClientService getProjectInstance(User user) throws Ex
                 return productionInstance;
             case "sandbox":
                 if(sandboxInstance == null)
-                    sandboxInstance = makeInstance("https://development.api.starkinfra.com/");
+                    sandboxInstance = makeInstance("https://sandbox.api.starkinfra.com/");
```

Comment:
Opaaa hahahah. Só um comentário mesmo. Quando for mexer aqui sempre adiciona uma nova porque se esquecer o SDK ainda funciona

Reply:
Droga kkkkk